openpype/hosts/fusion/plugins/publish/collect_existing_frames.py

```diff
@@ -0,0 +1,47 @@
+import pyblish.api
+import os
+
+
+class CollectFusionRenders(pyblish.api.InstancePlugin):
```

Comment:
```suggestion
class CollectFusionExpectedRenders(pyblish.api.InstancePlugin):
```
Class name must be unique!